### PR TITLE
Improving version information

### PR DESCRIPTION
**Pull request by mathersm-dst**
Added information about test compiler configurations and build dependency versions to the VERSION file and README.md

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1,14 @@
...
+---------------
+Visual Studio 2015
+Visual Studio 2017
+g++ 4.8.1
+g++ 6.2.0
+
+Dependencies:
+------------
+ACE-TAO version 6.4.3
+OpenDDS version 3.10
```
Comment (jwillemsen) [inline VERSION]: OpenDDS 3.11 is the latest release

Comment (mathersm-dst) [inline VERSION]: It wasn't used initially with our previous release, but will show up in the notes for 1.4.4

Comment (jwillemsen) [inline VERSION]: But than you should add a remark that OpenDDS support isn't supported with Visual Studio 2017 because 3.10 doesn't support that

Comment (mathersm-dst) [inline VERSION]: Good point, I'll update it now then to avoid confusion.

**Comment by jwillemsen** [comment]
Thinking more about this, isn't it easier to add information about a made release to the releases page on github (for example https://github.com/LASAGNE-Open-Systems/LASAGNE-Core/releases/tag/v1.4.3) and have the version file contain the current development dependencies?

**Comment by mathersm-dst** [comment]
Yes that would probably be a better way of doing it. Moving forward the version file will reflect the development dependencies. The initial values were merely a question of when the file itself gets introduced.

**Pull request merged**